### ISSUE TITLE
Create migrate to remove authors table

### ISSUE THREAD

**Issue by meltedspork** [deafchi/signsfive-api#17]
We are using Auth0's user database so there are no longer necessary for this.

CC: @kratsg

**Comment by meltedspork** [comment]
Are we removing whole table of `authors` ? @kratsg

**Comment by kratsg** [comment]
We are. Need to update IDs everywhere to use the right one from auth0 -- but some testing is required, etc...

**Comment by meltedspork** [comment]
i am thinking to keep this PR small. 

first PR is to remove and second PR is to add from auth0 

what can we take from auth0 in replace for?

**Comment by kratsg** [comment]
auth0 provides a user id. But you should look at what's in the tokens they give us :)

**Comment by meltedspork** [comment]
this is what we want to store? (not all of it of course) I am assuming - email, email_verfiied, family_name, given_name, groups, roles, and locale?
```{
    "profile": {
        "email": "j*********@****.com",
        "email_verified": true,
        "family_name": "Salerno",
        "gender": "male",
        "given_name": "Jason",
        "https://api.signsfive.com/groups": [],
        "https://api.signsfive.com/roles": [],
        "locale": "en-US",
        "name": "Jason Salerno",
        "nickname": "jms******",
        "picture": "https://scontent.xx.fbcdn.net/v/t1.0-1/p50x50/path-to-my-fb-profile-pic",
        "sub": "facebook|0987654321",
        "updated_at": "2018-02-11T23:39:11.527Z"
    }
}```

**Comment by kratsg** [comment]
Nope, we only need to store `sub`: `facebook|0987654321` in your case. It specifies the main log-in authenticator provider and the ID for that. It's unique to everyone who logs in, in any way. Everything else we can ping auth0 for if we need the extra information. The access token provided to the API will contain the permissions. So no extra look up needed.

**Comment by kratsg** [comment]
See https://tools.ietf.org/html/rfc7519#section-4.1.2 for details.
